app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -15,6 +20,7 @@
 
     Button buttonLogin, buttonSignup;
     EditText textEmail, textPass;
+    TextView termsAndConditions;
```

Comment:
Some things that I didn't mark until now so as not to add problems, but that we should start to take care of:
- The naming convention for members are like `mName` and for static `sName`, so here it should be `mTermsAndConditions` and so.
- Take care of the access to the members to take care. All this should be private.